src/coreclr/interop/platform.h

```diff
@@ -12,15 +12,21 @@
 #define _ASSERTE(x) assert((x))
 #endif
 
+#ifndef ARRAYSIZE
```

Comment:
This needs to be below the include of `Windows.h`.

Reply:
Ok